php/libraries/OutputWrapper.class.inc

```diff
@@ -0,0 +1,185 @@
+<?php declare(strict_types=1);
+/**
+ * This class is designed to assist in writing command-line interface tools
+ * for LORIS.
+ *
+ * It contains helper functions to format output, etc.
+ *
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ */
+class OutputWrapper
+{
+    // Colours associated with the bash shell.
+    // @see https://misc.flogisoft.com/bash/tip_colors_and_formatting
```

Comment:
ANSI escape sequences and colours predate bash by a long shot, and the wikipedia page (https://en.wikipedia.org/wiki/ANSI_escape_code) is probably a better reference than someone's blog.

Reply:
This isn't resolved

Reply:
I made them before but it looks like they got reverted during rebasing